Source/ARTPushChannel.m

```diff
@@ -146,26 +150,30 @@ - (void)unsubscribeClient:(void(^_Nullable)(ARTErrorInfo *_Nullable))callback {
     }];
 }
 
-- (void)listSubscriptions:(void(^)(ARTPaginatedResult<ARTPushChannelSubscription *> *_Nullable, ARTErrorInfo *_Nullable))callback {
-    [self listSubscriptions:nil callback:callback];
-}
+- (BOOL)listSubscriptions:(NSDictionary<NSString *, NSString *> *)params callback:(void(^)(ARTPaginatedResult<ARTPushChannelSubscription *> *_Nullable, ARTErrorInfo *_Nullable))callback error:(NSError *_Nullable *_Nullable)errorPtr {
```

Comment:
It has a `BOOL` because it's how ObjC handle errors. We should respect that for good Swift/ObjC interoperability. An ObjC method with an `NSError **` as last parameter and a BOOL return type will bridge to a Swift method that throw an error.